utils/docker_build/build.sh

```diff
@@ -6,10 +6,8 @@
 REMOTE_URL=https://github.com/secure-device-onboard/client-sdk.git
 REMOTE_BRANCH=master
 
-git clone http://github.com/intel/safestringlib.git
-cd safestringlib/ && make
-cd .. && rm -f CMakeCache.txt
-export SAFESTRING_ROOT=/home/sdouser/client-sdk/safestringlib
+rm -f CMakeCache.txt
+export SAFESTRING_ROOT=/home/sdouser/safestringlib
```

Comment:
is this still required?
because env is already set.

Reply:
No. Can be removed.

Reply:
@DukeDavis12  Could you test and repost?

Reply:
Yes.. tested and reposted.